docs/deployment/breaking-changes.md

```diff
@@ -4,53 +4,49 @@ This section lists breaking changes introduced in OpenCTI, per version starting
 
 Please follow the migration guides if you need to upgrade your platform. 
 
-## OpenCTI 6.2
-
-### Change to the observable "promote"  
-
-The API calls that promote an Observable to Indicator now return the created Indicator instead of the original Observable.
+## Breakdown per version
 
-**GraphQL API**
+This table regroups all the breaking changes introduced, with the corresponding version in which the change was implemented.
 
-* Mutation `StixCyberObservableEditMutations.promote` is now deprecated
-* New Mutation `StixCyberObservableEditMutations.promoteToIndicator` introduced
+| Change                                                           | Deprecated in | Changed in |
+|:-----------------------------------------------------------------|:--------------|:-----------|
+| [Promote Observable API](#change-to-the-observable-promote-API)  | 6.2           | 6.5        |
+| [SAML authentication parameters](#change-to-SAML-authentication) |               | 6.2        |
+| [Major changes to Filtering API](#new-filtering-API)             |               | 5.12       |
 
 
-**Client-Python API**
-
-* Client-python method `client.stix_cyber_observable.promote_to_indicator` is now deprecated
-* New Client-python method `client.stix_cyber_observable.promote_to_indicator_v2` introduced
-
+## OpenCTI 6.2
 
-!!! warning "Discontinued Support"
+### Deprecation
 
-    Please note that the deprecated methods will be permanently removed in OpenCTI 6.5.
+<a id="change-to-the-observable-promote-API"></a>
```

Comment:
why do you need a a tag ? links are generated

Reply:
actually I've been copy-pasting how it's done elsewhere. Anchor links are not generated it seems.